packs/dotnet/dotnet/alerts/MemoryUsage.yml

```diff
@@ -0,0 +1,30 @@
+---
+
+name: Memory Usage
+
+details: |+
+  This alert is triggered when Memory usage is above 0.35
+type: STATIC
+
+nrql:
+  query: "SELECT average(apm.service.memory.physical) * rate(count(apm.service.instance.count), 1 minute) / 1000 as memoryUsage FROM Metric WHERE appName = 'DotNetApplication'"
```

Comment:
That specific appName is unique to your application. We need to remove the WHERE clause or find a field that will work for all customers